masterlist.yaml

```diff
@@ -301,3 +305,6 @@ plugins:
       - <<: *patchAvailable
         subs: [ 'DarkerNights.esp' ]
         condition: 'active("Radiant Clouds and Fogs.esp") and active("DarkerNights.esp") and not active("DarkerNights-Radiant.esp")'
+  - name: 'Scrap Everything.esp'
+    msg: [ *doNotClean ]
```

Comment:
:facepalm: I just realised you're actually the author of the mod. That happens so rarely I didn't even think to check.